docs/config.c

```diff
@@ -1519,14 +1520,14 @@
 ** .pp
 ** This is the string that will follow a message which has been forwarded
 ** in the main body of a message (when $$mime_forward is unset).
-** For a full listing of defined \fCprintf(3)\fP-like sequences see
+** For a full listing of defined \fIprintf(3)\fP-like sequences see
 ** the section on $$index_format.  See also $$attribution_locale.
 */
 
 { "forward_decode", DT_BOOL, true },
 /*
 ** .pp
-** Controls the decoding of complex MIME messages into \fCtext/plain\fP when
+** Controls the decoding of complex MIME messages into \fItext/plain\fP when
```

Comment:
@g-branden-robinson  What would you use here?  Bold, italics, or roman?  I don't know.